### DISCUSSION TITLE
The information of Technical suitability

### DISCUSSION THREAD

**Discussion by fukustar** [bitcraze/discussions#249] [Q&A]
Hi, I would like to know about technical suitability of crazyflie and its peripheral equipments.

Now I have crazyflie2.1 and some decks.
But there are few information about techical suitability.

I found the marks on crazyflie 2.1's box and it had FCC (by USA), CE(by EU), and also by Japan. (I showed the picture.)
![image](https://user-images.githubusercontent.com/91782857/187844171-3acd95ba-b094-4174-8b49-6081b30698c7.png)

Also on the lighthouse box, I found CE (by EU)
But there are more decks and also crazyradio PA, but I cannot found the marks.

I showed the list which I want to know that they have technical suitability (and also which one(CE or FCC or Japan's)) or not.
-------------------------------
・Crazyradio PA
・Flow deck
・Lighthouse deck
------------------------------

**Comment by tobbeanton** [comment]
Due to the nature of our products (development/lab products) most of them are not stable enough to make the certification feasible. Therefor only the Crazyflie 2.1 is certified by FCC/CE/MIC, an no other product yet. Most countries allow for lab/development exemptions but from what we have learned this is much stricter in Japan.

**Reply by fukustar** [reply]
Thank you and I'm sorry to be late.

I understood the situation. But I would like you to ask one more.
You said that

> Most countries allow for lab/development exemptions but from what we have learned this is much stricter in Japan

So about "Crazyradio PA", are there any certified by FCC or CE ?

**Reply by tobbeanton** [reply]
No the Crazyradio PA is not FCC or CE certified but I believe it has been KC certified in Korea and for limited units (test certification) in Japan.

**Reply by fukustar** [reply]
Thank you！！

Are there any certificate or mark of KC on the box?

**Reply by tobbeanton** [reply]
The KC certification has been done by a Korean distributor [Clrobur](http://clrobur.com/en/sub0203.php). I think they add that to the box when they sell in Korea.
